core/src/main/java/io/questdb/griffin/engine/table/AsyncFilterAtom.java

```diff
@@ -116,6 +104,18 @@ public void initCursor() {
         }
     }
 
+    public int maybeAcquireFilter(int workerId, boolean owner, SqlExecutionCircuitBreaker circuitBreaker) {
+        if (perWorkerLocks == null) {
+            return -1;
+        }
+        if (workerId == -1 && owner) {
+            // Owner thread is free to use its own filter anytime.
```

Comment:
```suggestion
            // Owner thread is free to use its own private filter anytime.
```

Reply:
Changed in 20ba60c6b4d95ff2aec27a0f99f028032b891afe